recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,11 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  run:
+    - kraken ==0.10.5beta
```

Comment:
Sorry for not stopping to bother you :-). Why the strict dependency here? Can't we just depend on kraken, and let kraken-all have an independent version number? This way, when kraken is updated we don't always have to update kraken-all as well.